monai/transforms/utils.py

```diff
@@ -411,7 +411,7 @@ def weighted_patch_samples(
 
 
 def correct_crop_centers(
-    centers: List[Union[int, torch.Tensor]],
+    centers: Union[NdarrayOrTensor, List[NdarrayOrTensor]],
```

Comment:
Hi @Nic-Ma, I looked at the use of this `correct_crop_centers`, 

one is with a list input: https://github.com/Project-MONAI/MONAI/blob/523a04728a57719c5c598bd5708d211b28527cbc/monai/transforms/utils.py#L563-L564 the other is without:
https://github.com/Project-MONAI/MONAI/blob/523a04728a57719c5c598bd5708d211b28527cbc/monai/transforms/utils.py#L506-L508
any idea why there's the difference?

Reply:
Hi @wyli ,

Thanks for your great finding!
I think the code was developed before we fixed the perf issue of GPU cached indices, etc.:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/utils.py#L360
Actually, we don't need to convert to list of tensor and convert tensor to numpy. I submitted PR https://github.com/Project-MONAI/MONAI/pull/3652 to optimize the logic.
BTW, this PR looks good to me.

Thanks.